src/builder/QuarkBuilder.sol

```diff
@@ -152,41 +151,43 @@ contract QuarkBuilder {
             }
 
             assertSufficientPaymentTokenBalances(
-                actions, chainAccountsList, repayIntent.chainId, supplementalPaymentTokenBalance
+                actionsArray, chainAccountsList, repayIntent.chainId, supplementalPaymentTokenBalance
             );
         }
 
         // Merge operations that are from the same chain into one Multicall operation
-        (quarkOperations, actions) = QuarkOperationHelper.mergeSameChainOperations(quarkOperations, actions);
+        (quarkOperationsArray, actionsArray) =
+            QuarkOperationHelper.mergeSameChainOperations(quarkOperationsArray, actionsArray);
 
         // Wrap operations around Paycall/Quotecall if payment is with token
         if (payment.isToken) {
-            quarkOperations =
-                QuarkOperationHelper.wrapOperationsWithTokenPayment(quarkOperations, actions, payment, useQuotecall);
+            quarkOperationsArray = QuarkOperationHelper.wrapOperationsWithTokenPayment(
+                quarkOperationsArray, actionsArray, payment, useQuotecall
+            );
         }
 
         // Construct EIP712 digests
         EIP712Helper.EIP712Data memory eip712Data;
-        if (quarkOperations.length == 1) {
+        if (quarkOperationsArray.length == 1) {
```

Comment:
This whole code block was deleted from `main` in #51. I'm guessing that if you merge p2 into p1 that you're going to have merge conflicts.

But if you rebase, you should be able to just delete any changes you've made here to lines 170-183.

Reply:
Oh I see.  Thanks! I have rebased it so this change is now gone 👍